lib/pbench/cli/agent/commands/tools/clear.py

```diff
@@ -172,6 +180,7 @@ def callback(ctxt, _param, value):
     return click.option(
         "-g",
         "--group",
+        "--groups",
```

Comment:
It might be nice to *use* these new aliases: although testing all of `-g`, `--group` and `--groups` would be more a test of `click` than Pbench, it's probably good to have some ongoing validation that the `click.option` hasn't gotten messed up.

Reply:
I see `--group` and `--groups` scattered randomly; also both `--remote` and `--remotes`, but I don't see any use of `-g` or `-r` or `-n`, and "names" is always `-name`.

While I'm still uncertain at some level how much I care, something in the back of my head says I must care a bit since I've brought it up several times. 😆

Reply:
Oh, fine; since this really ought to be more `click` testing than Pbench testing, I'm just going to mark this resolved.